lifecycle/lifecycle/deployer/provision.py

```diff
@@ -57,15 +57,18 @@ def provision_job(
         family = create_job_family_if_not_exist(manifest.name)
         family_dto = job_family_model_to_dto(family)
 
-        runtime_env_vars = merge_env_vars(manifest.runtime_env, secret_runtime_env)
         build_env_vars = merge_env_vars(manifest.build_env, secret_build_env)
-        runtime_env_vars = hide_env_vars(runtime_env_vars, build_env_vars)
+        all_runtime_vars = merge_env_vars(manifest.runtime_env, secret_runtime_env)
+        runtime_env_vars = manifest.runtime_env or {}
```

Comment:
I'm thinking if naming could be more consistent here. Because if `build_env + secret_build_env = build_env_vars`, then
`runtime_env + secret_runtime_env` should equal `runtime_env_vars`..? But such name is used below. So maybe it's better to rename `build_env_vars` to `all_build_vars`.

Reply:
Good idea, will rename to `all_build_vars`